src/kayenta/report/detail/reportMetadata.tsx

```diff
@@ -130,24 +131,26 @@ const ReportMetadata = ({ run }: IReportMetadata) => {
 
   return (
     <section className="report-metadata">
-      <div className="horizontal space-between bottom">
-        {metadataGroups.map((group, index) => (
-          <div key={group.label || index}>
-            <Label label={group.label || ''} extraClass="label-lg" />
-            <ul className="list-unstyled list-inline">
-              {group.entries.map(e => (
-                <li key={e.label || index}>
-                  <Label label={e.label} />
-                  {e.getContent()}
-                </li>
-              ))}
-            </ul>
-          </div>
-        ))}
-        <div key="source">
-          <Label label="Source" extraClass="label-lgf" />
-          <SourceLinks />
+      {metadataGroups.map((group, index) => (
+        <div className={`group group-${group.label}`} key={group.label || index}>
+          <Label label={group.label || ''} extraClass="label-lg" />
+          <ul className="list-unstyled">
+            {group.entries.map(e => (
+              <li key={e.label || index}>
+                <Label label={e.label} />
+                {e.label == 'scope' ? (
```

Comment:
Feels like this would little more future-proof and easier to read if you added a `popover: boolean` flag on the interface (and checked for it here) rather than special-casing `'scope'`.

Reply:
Great call, thanks!

Reply:
very yes